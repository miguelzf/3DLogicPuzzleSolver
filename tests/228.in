- - - - - -
R - - - R -
- B C - - -
- - - - - L
- - - - - -
- - X - - B

- - X - - G
- - - - - -
- - C - - P
- G - - - X
- - - - - -
- - - - - -

Y - P X - -
- - - - - -
L - X - - -
X - - Y - -
- - - - - -
- - - - - -

