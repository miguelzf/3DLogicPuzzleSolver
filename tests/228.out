
Solution:
r  r  r  r  r  l  
R  b  b  b  R  l  
-  B  C  b  -  l  
c  c  c  b  b  L  
c  g  g  g  b  -  
c  g  X  g  b  B  
================
c  g  X  g  g  G  ||  Y  y  P  X  -  l  
c  g  -  l  l  l  ||  l  y  p  p  p  l  
c  g  C  l  p  P  ||  L  y  X  -  p  l  
c  G  c  l  p  X  ||  X  y  y  Y  p  l  
c  -  c  l  p  p  ||  p  p  p  p  p  l  
c  c  c  l  l  l  ||  l  l  l  l  l  l  
